src/hubmap/get-dataset-list.js

```diff
@@ -0,0 +1,52 @@
+import { getConfig } from '../util/common.js';
+
+const HUBMAP_TOKEN =
+  'AgqkKmq0JN50EpK02r1qklja26aPk7N54Jz0oYm243zN6njoW3UqCpWlNw2lldEYJ7mG7YVd5gXvmC5gjvYvtGd28k';
+// const HUBMAP_SEARCH_URL = 'HUBMAP_SEARCH_URL';
+const DEFAULT_HUBMAP_SEARCH_URL =
+  'https://search.api.hubmapconsortium.org/v3/portal/search';
+
+// https://portal.hubmapconsortium.org/browse/dataset/b6adb50fe8ab0d3d596a04b233d5c702.json
+
+async function downloadAllDatasetListing() {
+  const token = HUBMAP_TOKEN;
+  const searchUrl = DEFAULT_HUBMAP_SEARCH_URL;
+
+  fetch(searchUrl, {
```

Comment:
This is extremely similar to the code in hubmap/metadata.js. Lets update the code in there to support this use case instead of duplicating the logic. It will also avoid multiple pitfalls currently present in this code